integration-tests/.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Integration Type Checker",
+            "program": "${workspaceFolder}/node_modules/dtslint/bin/index.js",
```

Comment:
should this be the `npm test` script instead of directly trying to run `dtslint`?

Reply:
I tried that by following https://code.visualstudio.com/docs/nodejs/nodejs-debugging, but it seems very specific to the debug command. I've tried swapping it for `npm test`, but it just hangs. 

```
        {
            "type": "node",
            "request": "launch",
            "name": "npm test",
            "cwd": "${workspaceFolder}",
            "runtimeExecutable": "npm",
            "runtimeArgs": [
                "run-script", "test"
            ],
            "port": 9229
        },
```